include/seqan3/range/views/minimiser.hpp

```diff
@@ -0,0 +1,481 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Mitra Darvish <mitra.darvish AT fu-berlin.de>
+ * \brief Provides seqan3::views::minimiser.
+ */
+
+#pragma once
+
+#include <deque>
+
+#include <seqan3/range/concept.hpp>
+#include <seqan3/range/views/detail.hpp>
+
+namespace seqan3::detail
+{
+// ---------------------------------------------------------------------------------------------------------------------
+// minimiser_view class
+// ---------------------------------------------------------------------------------------------------------------------
+
+/*!\brief The type returned by seqan3::views::minimiser.
+ * \tparam urng_t The type of the underlying ranges, must model std::forward_range, the reference type must model
+ *                std::totally_ordered. The typical use case is that the reference type is the result of
+ *                seqan3::kmer_hash.
+ * \implements std::ranges::view
+ * \implements std::ranges::random_access_range
```

Comment:
remove this as the view does not implement a random access range. Same below with sized range.